crates/mipsy_interactive/src/interactive/commands/mod.rs

```diff
@@ -37,25 +35,27 @@ pub(crate) use step2syscall::step2syscall_command;
 
 use super::{error::CommandResult, State};
 
+// TODO(joshh): remove once if-let chaining is in
+#[derive(Clone)]
 pub(crate) enum Arguments {
     Exactly { required: Vec<String>, optional: Vec<String> },
     VarArgs { required: Vec<String>, format: String, },
 }
 
+// TODO(joshh): remove once if-let chaining is in
```

Comment:
IMO: no todo's in comments, make an issue

Reply:
![Screenshot_2022-07-27-19-46-25-413_com android chrome](https://user-images.githubusercontent.com/28849752/181217421-d6b50a85-e6ee-4d69-99e7-0219e737bc9a.png)
pwned